src/uniref/adapters/uniRefConverter.ts

```diff
@@ -70,7 +64,7 @@ export type UniRefLiteAPIModel = {
   organismCount: number;
   representativeId: string;
   seedId: string;
-  memberIdTypes: MemberIdType;
+  memberIdTypes?: string[];
```

Comment:
Not sure why the change to remove the specificity and make it a string. Aren't we supposed to only have 3 possible options here?

Reply:
Not sure either. Updated to `EntryType`